integration_tests/inttest_lib/testing_stand_executor.py

```diff
@@ -172,6 +172,10 @@ def await_workloads(self):
             runner.await_completion()
 
     def conclude(self):
+        if types.LAMBDA_FUNCTIONS in self.resource_types_to_run:
```

Comment:
this part will be removed and considered in the corresponding await Lambda method (will be added in a separate PR)